platform/install/environments/gcp.mdx

```diff
@@ -0,0 +1,292 @@
+Privacy Statement: https://www.loft.sh/legal/privacy
+? Please specify an email address for the admin user
+```
+
+:::tip retry
+if the command takes to long to execute, for example due to other cluster
+operations, simply run the command one more time
+:::
+</Step>
+
+<Step>
+Connect to the platform.
+
+Once the platform is deployed, default browser with the platform UI opens and you should see the output similar to this:
+
+```bash title="platform deployment output"
+##########################   LOGIN   ############################
+
+Username: admin
+Password: 9758c908-b931-4edd-b3cb-3f034e50651a  # Change via UI or via: vcluster platform reset password
+
+Login via UI:  https://hyx4907.loft.host
+Login via CLI: vcluster login https://hyx4907.loft.host
+
+#################################################################
+
+vCluster Platform was successfully installed and can now be reached at: https://hyx4907.loft.host
+
+Thanks for using vCluster Platform!
+19:34:46 done You are successfully logged into vCluster Platform!
+- Use `vcluster platform create vcluster` to create a new virtual cluster
+- Use `vcluster platform add vcluster` to add an existing virtual cluster to a vCluster platform instance
+```
+
+When logging in via UI, provide: <Label>First Name</Label>, <Label>Last Name</Label>, <Label>Email</Label> (should default to
+the one supplied earlier) and <Label>Organization</Label>.
+
+To login via CLI, simply copy/pasted the `Login via CLI` command.
+
+This is the basic platform deployment, from there you can refer to the
+#post-install-steps to start using the platform. However there are a few
+additional configuration steps that can be done.
+</Step>
+
+</Flow>
+
+<Flow>
+
+### Expose platform UI via load balancer
+
+<Step>
+Create a LoadBalancer service to expose the platform UI.
+
+:::note
+Please note that this
+assumes the platform is deployed in the `vcluster-platform` namespace which is a
+default deployment namespace.
+:::
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: vcluster-platform-loadbalancer
+  namespace: vcluster-platform
+spec:
+  type: LoadBalancer
+  externalTrafficPolicy: Cluster
+  selector:
+    app: loft
+  ports:
+  - name: https
+    protocol: TCP
+    port: 443
+    targetPort: 10443
+EOF
+```
+</Step>
+
+<Step>
+Once the service is active, obtain the external IP address:
+
+```bash
+kubectl get svc vcluster-platform-loadbalancer -n vcluster-platform
+```
+
+and navigate to the IP address in your browser `https://<EXTERNAL_IP>`.
+
+:::note certificate
+Note that the platform uses a self-signed certificate, so you need to
+accept the warning in your browser. For production use, you should replace the
+certificate with a valid one.
+:::
+
+To learn more about how to configure certificate and TLS, read the [TLS
+configuration guide](/platform/configure/domain#configure-tls)
+</Step>
+
+</Flow>
+
+<Flow>
+
+### Setup custom domain and configure DNS
+
```

Comment:
This step feels that this should be described as an option earlier in the deployment process of the platform so that you aren't deploying the platform and then having to re-deploy.

Similarly, we should provide options that if you already have a platform, here's how you would configure it and re-deploy.